webview-ui/src/components/welcome/RooTips.tsx

```diff
@@ -0,0 +1,92 @@
+import { VSCodeLink } from "@vscode/webview-ui-toolkit/react"
+import { useTranslation } from "react-i18next"
+import { useState, useEffect } from "react"
+import clsx from "clsx"
+
+const tips = [
+	{
+		icon: "codicon-list-tree",
+		href: "https://docs.roocode.com/features/boomerang-tasks",
+		titleKey: "rooTips.boomerangTasks.title",
+		descriptionKey: "rooTips.boomerangTasks.description",
+	},
+	{
+		icon: "codicon-pinned",
+		href: "https://docs.roocode.com/basic-usage/using-modes",
+		titleKey: "rooTips.stickyModels.title",
+		descriptionKey: "rooTips.stickyModels.description",
+	},
+	{
+		icon: "codicon-tools",
+		href: "https://docs.roocode.com/basic-usage/how-tools-work",
+		titleKey: "rooTips.tools.title",
+		descriptionKey: "rooTips.tools.description",
+	},
+]
+
+interface RooTipsProps {
+	cycle?: boolean
+}
+
+const RooTips = ({ cycle = false }: RooTipsProps) => {
+	const { t } = useTranslation("chat")
+	const [currentTipIndex, setCurrentTipIndex] = useState(Math.floor(Math.random() * tips.length))
+	const [isFading, setIsFading] = useState(false)
+
+	useEffect(() => {
+		if (!cycle) return
+
+		const intervalId = setInterval(() => {
+			setIsFading(true) // Start fade out
+			setTimeout(() => {
+				setCurrentTipIndex((prevIndex) => (prevIndex + 1) % tips.length)
+				setIsFading(false) // Start fade in
+			}, 1000) // Fade duration
+		}, 11000) // 10s display + 1s fade
+
+		return () => clearInterval(intervalId) // Cleanup on unmount
+	}, [cycle])
+
+	const currentTip = tips[currentTipIndex]
+	const topTwoTips = tips.slice(0, 2)
+
+	return (
+		<div
+			className={clsx(
+				"flex flex-col items-center justify-center px-5 py-2.5 gap-4",
+				cycle && "h-[5em] overflow-visible m-5",
+			)}>
+			{/* If we need real estate, we show a compressed version of the tips. Otherwise, we expand it. */}
+			{cycle ? (
+				<>
+					<div className="opacity-70 pb-1"> Did you know about...</div>
```

Comment:
I think we need to internationalize this

Reply:
This is gone, there's no active 'cycle=true' version in the UI right now. But yeah, if we bring it back we should internationalize.

Reply:
(It's technically dead code right now and should be gone altogether but I think cycling tips should be something explored still, tbh.)